bin/update

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
Many of these files were generated after running `rake rails:update`.  It was interactive and I was able to say `n` perhaps I should've done that?

[see 1.4 in the guide to upgrading Rails](http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#the-update-task)